test/fhirtypes/Period.test.ts

```diff
@@ -1,44 +1,43 @@
-import { Period, validatePeriod } from '../../src/fhirtypes/Identifier';
-import { InvalidPeriodError } from '../../src/errors/InvalidPeriodError';
+import { InvalidPeriodError } from '../../src/errors';
+import { Period, validatePeriod } from '../../src/fhirtypes';
 
 describe('Period', () => {
-  let earlier: Date;
-  let later: Date;
+  let dateOnly: string;
+  let hasTimeZone: string;
 
   beforeAll(() => {
-    earlier = new Date();
-    later = new Date();
-    later.setFullYear(later.getFullYear() + 1);
+    dateOnly = '2006-03-29';
+    hasTimeZone = '2010-02-07T13:28:17+03:00';
```

Comment:
Even here, even though these are re-used, they're so trivial that I wonder if it's better just to put the string literals right in the tests?  As it is now, when I'm reading the tests, I need to remember what those variables represent and which one comes before the other (chronologically).  That's all very obvious, however, if the test just uses string literals.

The other approach, which is closer to what you had before, is to give the variables more meaningful names so it's clear what they are and how they relate to each other -- for example `earlyDate` and `laterDateTime`.